stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -72,6 +200,127 @@ public View getViewForHighlighting(Object element) {
     return (View)element;
   }
 
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+    for (Method method : getViewGetters()) {
+      boolean caught = false;
+      try {
+        @Nullable ViewDebug.ExportedProperty annotation = getExportedAnnotation(method);
+        final Object returnedValue = method.invoke(element);
+        final String cssName = method.getName(); //convertGetterNameToCssName(method.getName());
+
+        if (method.getName().equals("getId")) {
+          @Nullable String id = getIdAttribute(element);
+
+          if (id == null) {
+            styles.store("id", "(none)", false);
```

Comment:
we should put these string constants into actual string constants, e.g.

```
styles.store(ID_STYLE_NAME, ID_STYLE_VALUE_NONE, false);
```

etc.